packages/rest/package.json

```diff
@@ -26,8 +26,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "class-transformer": "^0.4.0",
-    "class-validator": "^0.13.1",
+    "class-transformer": "0.5.1",
+    "class-validator": "^0.13.2",
```

Comment:
I was running into issues with class-validator 0.13.2, so I pinned it to `0.13.1` in AFJ:

https://github.com/typestack/class-validator/issues/1412